commands/run_migration.sh

```
@@ -2,7 +2,7 @@
 
 # SQLAlchemy migrate
 ALEMBIC_CONFIG="/usr/src/alembic/alembic.ini"
-MIGRATIONS_DIR="/usr/src/fastapi/database/migrations/versions"
+MIGRATIONS_DIR="/usr/src/app/database/migrations/versions"
```

This line contains a stray review marker `/++5++/` appended after the variable assignment. That token is not valid shell syntax and will break the script. Remove the `/++5++/` sequence so the line is a plain assignment, for example:

MIGRATIONS_DIR="/usr/src/app/database/migrations/versions"

Also consider quoting variables when used later (e.g. `alembic -c "$ALEMBIC_CONFIG"`) and adding `set -e` at the top of the script to fail fast on errors.